Build/NuGet/Microsoft.ChakraCore.nuspec

```diff
@@ -4,15 +4,15 @@
     <id>Microsoft.ChakraCore</id>
     <!-- Note: actual version number is overridden by the NuGet package creation command. -->
     <version>$version$</version>
-    <authors>Chakra Team</authors>
+    <authors>Microsoft</authors>
     <owners>Chakra Team</owners>
     <licenseUrl>https://github.com/Microsoft/ChakraCore/blob/master/LICENSE.txt</licenseUrl>
     <projectUrl>https://github.com/Microsoft/ChakraCore</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <developmentDependency>true</developmentDependency>
     <description>ChakraCore is the core part of the Chakra Javascript engine that powers Microsoft Edge.</description>
     <releaseNotes>https://github.com/Microsoft/ChakraCore/wiki/Roadmap#release-notes</releaseNotes>
-    <copyright>Copyright (C) 2016 Microsoft</copyright>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
```

Comment:
Doesn't this non-ASCII character cause issues sometimes (and thus (C) is preferable)?

Reply:
I'd go with 'probably', but this was mandated to work with external processes.